VCFv4.3.tex

```diff
@@ -563,15 +586,14 @@ \section{INFO keys used for structural variants}
 ##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
 \end{verbatim}
 \normalsize
-\end{samepage}
```

Comment:
This line has to stay. The latex build is failing because there's an unmatched `\begin{samepage}`.